enhancements/installer/gcp-dns-creation-specified-project.md

```diff
@@ -0,0 +1,223 @@
+  - "@patrickdillon"
+  - "@barbacbd"
+  - "@sadasu"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - "@joelspeed"
+creation-date: 2025-06-16
+last-updated: 2025-06-16
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - "https://issues.redhat.com/browse/CORS-4012"
+superseded-by:
+  - "/enhancements/installer/gcp-ipi-shared-vpc.md"
+---
+# GCP DNS Creation Specified Project
+
+## Summary
+
+The enhancement adds the capability to deploy a GCP XPN ([shared vpc]((https://cloud.google.com/vpc/docs/shared-vpc))) cluster where the DNS management can be in a
+separate service project following Google's suggested architecture. The enhancement adds the capability to specify
+a service project (for the main installation), host project (for network resources), and a [possible] third project 
+specifically for DNS zones and records.
+
+## Motivation
+
+The GCP suggested architecture states that placing DNS management in a separate (third) project is recommended for
+improved security, organization, and management of DNS resources.
+
+Dedicating a separate project to DNS establishes a centralized location for managing all DNS zones and records. The
+separation of resources also increases security by limiting access to DNS resources. Granular permissions can be
+granted to those that are required to work with DNS resources. Separating these resources to another project _could_ 
+promote lower spending through lower project usages. 
+
+### User Stories
+
+- As a user I want to deploy OCP on GCP XPN within 3 projects; a host project, a service
+  project to deploy openshift, and a service project where DNS zones can be managed.
+
+### Goals
+
+- Enable users to deploy GCP clusters with Shared VPC networking using the IPI
+  workflow, where users can specify a third project where DNS will be managed.
+- Enable users to deploy GCP clusters with Shared VPC networking using the UPI
+  workflow, where users can specify a third project where DNS will be managed.
+- Enable users to alternatively provide pre-provisioned resources to enable
+  minimal permission requirements in the Service Project for DNS management. This
+  includes the names of the private DNS Zone.
+
+### Non-Goals
+
+- Provisioning and/or configuring the VPC, Network, and Subnetworks are to be
+  done manually by the user prior to running the installer.
+- Provisioning and/or configuring the public DNS zone. When the user specifies the
+  project for the private DNS Zone, the public zone must exist in the same project
```

Comment:
@barbacbd Shall we update such statements according to the current behavior, i.e. the public DNS zone will be in the cluster's service project, rather then in the same project as the private DNS zone?